### ISSUE TITLE
Docker Scout

### ISSUE THREAD

**Issue by DaanSelen** [shuricksumy/docker-wgdashboard#29]
Hi Shuricksumy! Good work on your image, I like seeing your work.

Do you scan you images with docker scout?

**Comment by shuricksumy** [comment]
Hi,
I have not scanned yet.

It looks like a good idea,
Thanks, will try to do this.

**Comment by DaanSelen** [comment]
If you need help, let me know!

**Comment by shuricksumy** [comment]
> If you need help, let me know!

Thanks for that

I play a bit with that and got report
https://github.com/shuricksumy/docker-wgdashboard/actions/runs/11515382864#summary-32055938916

Could you please check this. If this is ok for understanding of possible issues ?

Than I will try to improve this somehow to see this reports on main readme pages or something like that.

**Comment by shuricksumy** [comment]
Added **Docker Scout**
- in Docker Hub https://hub.docker.com/repository/docker/shuricksumy/wgdashboard/general
- in GitHub as regular job https://github.com/shuricksumy/docker-wgdashboard/actions/workflows/scout-scan.yml

**Comment by DaanSelen** [comment]
> > If you need help, let me know!
> 
> Thanks for that
> 
> I play a bit with that and got report https://github.com/shuricksumy/docker-wgdashboard/actions/runs/11515382864#summary-32055938916
> 
> Could you please check this. If this is ok for understanding of possible issues ?
> 
> Than I will try to improve this somehow to see this reports on main readme pages or something like that.

The report is no longer available. I can't see it.

**Comment by DaanSelen** [comment]
Overall if possible try to reduce all vulnerabilities. In my opinion

**Comment by shuricksumy** [comment]
> > > If you need help, let me know!
> > 
> > 
> > Thanks for that
> > I play a bit with that and got report https://github.com/shuricksumy/docker-wgdashboard/actions/runs/11515382864#summary-32055938916
> > Could you please check this. If this is ok for understanding of possible issues ?
> > Than I will try to improve this somehow to see this reports on main readme pages or something like that.
> 
> The report is no longer available. I can't see it.

Yes, it was deleted - here the latest
https://github.com/shuricksumy/docker-wgdashboard/actions/runs/11516841111

**Comment by shuricksumy** [comment]
> Overall if possible try to reduce all vulnerabilities. In my opinion
2 high priority issues for Python - will see what I can do here
![image](https://github.com/user-attachments/assets/a243a9b2-190a-4805-a1d9-197ad96649f3)
![image](https://github.com/user-attachments/assets/d9e04b48-cee6-4b31-b6de-60e09bd92d1e)

**Comment by shuricksumy** [comment]
I changed docker image to Alpine linux as it has fixed issues. 

It looks like there is only issue connected to the original application. I checked the approach how they analyze it and realized that only critical issues are important for them. So I will continue generating reports for my docker images just be sure that there is no issue on linux/docker side but hope all issues connected to application will be closed in application code

 
![image](https://github.com/user-attachments/assets/3a1d7ca6-5c85-4d40-83c1-7d587ad995ea)
